projects/plugins/jetpack/modules/markdown/class-wpcom-markdown.php

```diff
@@ -270,13 +298,12 @@ public function sanitize_setting( $input ) {
 
 	/**
 	 * Prints HTML for the Writing setting
-	 * @return null
 	 */
 	public function post_field() {
 		printf(
 			'<label><input name="%s" id="%s" type="checkbox"%s /> %s</label><p class="description">%s</p>',
-			self::POST_OPTION,
-			self::POST_OPTION,
+			esc_attr( self::POST_OPTION ),
+			esc_attr( self::POST_OPTION ),
 			checked( $this->is_posting_enabled(), true, false ),
 			esc_html__( 'Use Markdown for posts and pages.', 'jetpack' ),
 			sprintf( '<a href="%s">%s</a>', esc_url( $this->get_support_url() ), esc_html__( 'Learn more about Markdown.', 'jetpack' ) )
```

Comment:
We could take that opportunity to switch to numbered placeholders, that would be cleaner, we wouldn't have to repeat that line?

Reply:
Gave that a shot! Let me know if I did that correctly.